src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -2580,7 +2618,9 @@ export class PatientViewPageStore {
             getServerConfig().fhirspark!.port !== 'undefined'
         )
             port = ':' + getServerConfig().fhirspark!.port;
-        return '//' + host + port + '/mtb/' + id;
+        return (
+            'http://' + host + port + (followUp ? '/followup/' : '/mtb/') + id
```

Comment:
Did the http here just slip through from testing?

Reply:
This is a good catch! This should definitely be changed before merging, as production sites *should* use https, so a protocol neutral implementation should be preferred.

Reply:
Yes, unfortunately this did slip through and wasn't intentional. Should also be fixed now.